clj-refactor.el

```diff
@@ -1058,10 +1058,13 @@ word test in it and whether the file lives under the test/ directory."
     (when (and cljr-add-ns-to-blank-clj-files
                (cljr--clojure-ish-filename-p (buffer-file-name))
                (= (point-min) (point-max)))
-      (clojure-insert-ns-form)
-      (newline 2)
-      (when (cljr--in-tests-p)
-        (cljr--add-test-declarations)))))
+      ;; TODO: replace (if ...) with (cider-expected-ns) once dependable
```

Comment:
If `cider-expected-ns` now falls back to `clojure-expected-ns` this `if`, and the todo, can be removed.  Our next release will be right after cider 0.11, I think.